sql/moz-fx-data-shared-prod/firefox_desktop/baseline_active_users/view.sql

```diff
@@ -2,20 +2,33 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.firefox_desktop.baseline_active_users`
 AS
 SELECT
-  * EXCEPT (app_display_version, normalized_channel, normalized_os, normalized_os_version) REPLACE(
+  * EXCEPT (
+    app_display_version,
+    normalized_channel,
+    normalized_os,
+    normalized_os_version,
+    distribution_id
+  ) REPLACE(
     IFNULL(country, '??') AS country,
     IFNULL(city, '??') AS city,
     COALESCE(REGEXP_EXTRACT(locale, r'^(.+?)-'), locale, NULL) AS locale
   ),
   CASE
     WHEN LOWER(IFNULL(isp, '')) = 'browserstack'
       THEN CONCAT('Firefox Desktop', ' ', isp)
-    WHEN LOWER(IFNULL(distribution_id, '')) = 'mozillaonline'
-      THEN CONCAT('Firefox Desktop', ' ', distribution_id)
+    WHEN LOWER(
+        IFNULL(COALESCE(last_seen.distribution_id, distribution_mapping.distribution_id), '')
+      ) = 'mozillaonline'
+      THEN CONCAT(
+          'Firefox Desktop',
+          ' ',
+          COALESCE(last_seen.distribution_id, distribution_mapping.distribution_id)
+        )
     ELSE 'Firefox Desktop'
   END AS app_name,
   app_display_version AS app_version,
   normalized_channel AS channel,
+  COALESCE(last_seen.distribution_id, distribution_mapping.distribution_id) AS distribution_id,
```

Comment:
@irrationalagent should we also have a value to identify if the distribution_id came from baseline or from the mapping table?

Reply:
yes i suppose a distribution_id_source with values `legacy` or `glean` makes sense.

Reply:
Just added. Thanks!